AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/editor_physx_collider.py

```diff
@@ -0,0 +1,499 @@
+
+        Used to set the PhysX Collider's Trigger Property.
+        """
+        self.component.set_component_property_value(self.Path.TRIGGER, value)
+
+    def get_is_trigger(self) -> bool:
+        """
+        Property Type, Default Visibility - ('bool', 'Visible')
+
+        Used to get the PhysX Collider's Trigger Property.
+        """
+        return self.component.get_component_property_value(self.Path.TRIGGER)
+
+    def set_is_simulated(self, value: bool) -> None:
+        """
+        Property Type, Default Visibility - ('bool', 'Visible')
+
+        Used to set the PhysX Collider's Simulated Property value.
+        """
+        self.component.set_component_property_value(self.Path.SIMULATED, value)
+
+    def get_is_simulated(self) -> bool:
+        """
+        Property Type, Default Visibility - ('bool', 'Visible')
+
+        Used to get the PhysX Collider's Simulated Property value.
+        """
+        return self.component.get_component_property_value(self.Path.SIMULATED)
+
+    def set_in_scene_queries(self, value: bool) -> None:
+        """
+        Property Type, Default Visibility - ('bool', 'Visible')
+
+        Used to set the PhysX Collider's In Scene Queries Property value.
+        """
+        self.component.set_component_property_value(self.Path.IN_SCENE_QUERIES, value)
+
+    def get_in_scene_queries(self) -> bool:
+        """
+        Property Type, Default Visibility - ('bool', 'Visible')
+
+        Used to get the PhysX Collider's In Scene Queries Property value.
+        """
+        return self.component.get_component_property_value(self.Path.IN_SCENE_QUERIES)
+
+    def set_offset(self, x: float, y: float, z: float) -> None:
+        """
+        Property Type, Default Visibility - ('Vector3', 'Visible')
+
+        Used to set the PhysX Collider's Offset Property.
+        """
+        _validate_xyz_is_float(x, y, z, f"One of the offset inputs are not a float - X: {x}, Y: {y}, Z {z}")
+
+        self.component.set_component_property_value(self.Path.OFFSET, math.Vector3(x, y, z))
+
+    def get_offset(self) -> math.Vector3:
+        """
+        Property Type, Default Visibility - ('Vector3', 'Visible')
+
+        Used to get the PhysX Collider's Offset Property value.
+        """
+        return self.component.get_component_property_value(self.Path.OFFSET)
+
+    def set_rotation(self, x: float, y: float, z: float) -> None:
+        """
+        Property Type, Default Visibility - ('Quaternion', 'Visible')
+
+        Used to set the PhysX Collider's Rotation Property.
+        """
+        _validate_xyz_is_float(x, y, z, f"One of the rotation inputs are not a float - X: {x}, Y: {y}, Z {z}")
+
+        rotation = math.Quaternion()
+        rotation.SetFromEulerDegrees(math.Vector3(x, y, z))
+        self.component.set_component_property_value(self.Path.ROTATION, rotation)
+
+    def get_rotation(self) -> math.Vector3:
+        """
+        Property Type, Default Visibility - ('Vector3', 'Visible')
```

Comment:
does this return an azlmbr.math.Quaternion since that's what's being set?

Reply:
Good catch.